core/components/organisms/calendar/__tests__/Calendar.test.tsx

```diff
@@ -242,3 +242,29 @@ describe('Calendar compoennt', () => {
     expect(FunctionValue).toHaveBeenCalled();
   });
 });
+
+describe('text color for different states', () => {
+  it('should have the text color as white when the state is active', () => {
+    const { getByTestId } = render(<Text color="white">{'Design System'}</Text>);
+    expect(getByTestId('DesignSystem-Text')).toHaveClass('color-white');
+  });
+
```

Comment:
Please write valid test cases for Calendar component instead of writing redundant test cases for Text component